### PR TITLE
Add lighthouse github action

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Who is this for?
People who want a performant, usable site, without having to drain their data budgets.

# What is this helping with
Currently, when viewing `/works`, which does all of nothing much but give information, we load in 3.69mb of content. 

Screenshot 2020-02-19 at 08 43 38

This is completely inaccessible and irresponsible.

We have the bundle sizes, but those seem to be good for diagnosing where the problem is, and this should allow us to set and stick to budgets.

I assume this creep has happened as we haven't paid much attention to it, but I would suggest it is now time.

### CODE REVIEW

File: budgets.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[{
+    "resourceSizes": [{
+            "resourceType": "script",
+            "budget": 1000
```
Comment (jamesgorrie) [inline budgets.json]: These are way over what they should be and placeholders.

Comment (gestchild) [reply budgets.json]: Is this the transferred size of the files or the unpacked size?

Comment (jamesgorrie) [reply budgets.json]: [Transfer size](https://developers.google.com/web/tools/lighthouse/audits/budgets)

**Comment by kenoir** [comment]
@jamesgorrie This has been open for too long, can it be closed or merged?

**Comment by kenoir** [comment]
Closing as 💀

**Pull request closed without merging**